Sources/Spacing/Spacing.swift

```diff
@@ -0,0 +1,42 @@
+import CoreFoundation
+
+extension Warp {
+    public enum Spacing: CGFloat {
+        /// Separation of 2 points.
+        case spacing25 = 2
```

Comment:
Just a heads up that these does not exist on WARP as of yet so I will deprecate these when we have a source of truth on Figma.
But as long as we don't have that feel free to add anything you need with the structure that makes sense to you here 👍

Reply:
I actually just added it because android have it 🙈

Reply:
😅👍 It should be great, now at least we have a place and structure for it 🥳 we will fill them with correct numbers when WARP is ready.